openmetadata-ui/src/main/resources/ui/src/utils/CustomizableLandingPageUtils.tsx

```diff
@@ -55,13 +55,17 @@ export const getAddWidgetHandler =
     if (
       placeholderWidgetKey === LandingPageWidgetKeys.EMPTY_WIDGET_PLACEHOLDER
     ) {
+      const emptyWidgetPlaceholder = currentLayout.find(
+        (item) => item.i === LandingPageWidgetKeys.EMPTY_WIDGET_PLACEHOLDER
+      ) ?? { x: 0, y: 99 };
+
```

Comment:
will it still work as expected if I have multiple EmptyWidgets? 
Or are we not marking the removed widget as EmptyWidget

Reply:
yes, it will work as the empty widgets are still set with the previous widget which was removed. so even we have multiple empty widget It will work fine. You can refer to the video as well.
<img width="347" alt="image" src="https://github.com/user-attachments/assets/312af43f-99ad-455c-b775-e22b4bd8da2d">